nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/entity/ProcessorRunStatusEntity.java

```diff
@@ -26,6 +26,9 @@
 @XmlType(name = "processorRunStatus")
 public class ProcessorRunStatusEntity extends ComponentRunStatusEntity {
 
+    // SUPPORTED_STATE *MUST* match the allowableValues below
+    // if it doesn't, OpenAPI/Swagger generation will generate
+    // correct clients, and they will have errors when deserializing
```

Comment:
I think a simpler but completely specific reminder comment would be better.

All we really want is to make sure when the constant changes, the annotation is updated as well. The rest is technical detail, one that could be written to almost all similar places.

I'd do something like this:

```suggestion
    // Must match the @ApiModelProperty.allowableValues above getState()
```